src/App.js

```diff
@@ -1,5 +1,122 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+
+export const ERROR_MESSAGES = {
+  EMPTY_CAR_NAMES: "[ERROR]: 빈 문자열은 입력할 수 없습니다.",
+  DUPLICATE_CAR_NAME: "[ERROR]: 같은 이름은 불가능합니다.",
+  INVALID_CAR_NAME_LENGTH: "[ERROR]: 이름은 5자 이하만 가능합니다.",
+  INVALID_ATTEMPT_COUNT: "[ERROR]: 숫자만 입력 가능합니다.",
+  ZERO_ATTEMPT_COUNT: "[ERROR]: 0은 입력할 수 없습니다.",
+  NEGATIVE_ATTEMPT_COUNT: "[ERROR]: 음수는 입력할 수 없습니다.",
+  NON_INTEGER_ATTEMPT_COUNT: "[ERROR]: 정수가 아닌 수는 입력할 수 없습니다.",
+};
 class App {
-  async run() {}
+  async run() {
+    const raceCars = await this.getRaceCars();
+    const attemptCount = await this.getAttemptCount();
+
+    this.startRace(raceCars, attemptCount);
+    this.printFinalResult(raceCars);
+  }
+
+  async getRaceCars() {
+    const inputCarNames = await Console.readLineAsync(
+      "경주할 자동차 이름을 입력하세요.(이름은 쉽표(,) 기준으로 구분)\n"
+    );
+    this.validateCarNames(inputCarNames);
+
+    const carNames = inputCarNames.split(",");
```

Comment:
공백 때문에 같은 이름을 다르게 인식 할 것 같아서 공백 제거하고 유효성 검사하면 좋을 것 같아요 
```suggestion
    const carNames = inputCarNames.split(",").map(name => name.trim());
```

Reply:
그 부분을 고려하지 못했던 것 같아요!
감사합니다!